buf.yaml

```diff
@@ -0,0 +1,19 @@
+build:
+  roots:
+    - .
+  excludes:
+    - vendor/github.com/tendermint
+lint:
+  use:
+    - DEFAULT
+    - COMMENTS
+    - FILE_LOWER_SNAKE_CASE
+  except:
+    - UNARY_RPC
+    - COMMENT_FIELD
```

Comment:
I would actually lean towards requiring field comments, but maybe it will make migration slower...

Reply:
We can, but it'll be super tedious. We don't even do this for standard go types.